### ISSUE TITLE
Play voice button does not work editing team

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#603]
```
What steps will reproduce the problem?
1. Edit a team.
2. Press the play button next to Voice.

What is the expected output? What do you see instead?

The button animates but no sound outputs?

What version of the product are you using? On what operating system?

Windows 7 64bit. Doesn't work with 0.9.19-r8672 or 0.9.19-r8806.
Works fine however in 0.9.18.
```

Original issue reported on code.google.com by `jimithebear` on 11 Apr 2013 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
Somehow I'm betting this is physfs again.  Tentatively tagging unc0rr.
Although, I can't rule out that it was my playing around in SDL audio.

I'll take a look tonight.

```

Original comment by `kyberneticist@gmail.com` on 12 Apr 2013 at 10:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `unC0Rr` on 15 Apr 2013 at 7:28
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue is back with r9047. Also affects r8963.

Issue 609 is similar although I never hear a sound when pressing the button.
```

Original comment by `jimithebear` on 30 May 2013 at 1:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `vittorio...@gmail.com` on 30 May 2013 at 8:44
* Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
```
Issue 609 has been merged into this issue.
```

Original comment by `kyberneticist@gmail.com` on 2 Jan 2014 at 9:43

**Comment by GoogleCodeExporter** [comment]
```
This might be fixed now.
Possibly by r9577634bd284 or r9577634bd284 or more likely r6505805e7f98

I can't remember but betting on r6505805e7f98

Anyway, going to mark fixed, 'cause I'm pretty sure it is, but someone please 
test and let me know

```

Original comment by `kyberneticist@gmail.com` on 2 Jan 2014 at 9:46
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This unfortunately is still not fixed as of 0.9.21 beta.
```

Original comment by `jimithebear` on 10 Dec 2014 at 2:41
